kf_lib_data_ingest/common/pandas_utils.py

```diff
@@ -289,12 +290,30 @@ def merge_wo_duplicates(left, right, left_name=None, right_name=None,
     left_name = left_name or 'Left'
     right_name = right_name or 'Right'
 
+    # Check if merge col(s) are present in DataFrame
+    def check_merge_col(merge_col, df_name, df, err_msgs):
+        if merge_col not in df.columns:
+            err_msgs.append(
+                f"'{merge_col}' not found in {df_name}: {df.columns}"
+            )
+        return err_msgs
+
     err = []
-    oncol = kwargs['on']
-    if oncol not in left.columns:
-        err.append(f"'{oncol}' not found in {left_name}: {left.columns}")
-    if oncol not in right.columns:
-        err.append(f"'{oncol}' not found in {right_name}: {right.columns}")
+    if 'on' in kwargs:
+        merge_col = kwargs['on']
+        err = check_merge_col(merge_col, left_name, left, err)
+        err = check_merge_col(merge_col, right_name, right, err)
+    elif ('left_on' in kwargs) and ('right_on' in kwargs):
+        merge_col = kwargs['left_on']
+        err = check_merge_col(merge_col, left_name, left, err)
+        merge_col = kwargs['right_on']
+        err = check_merge_col(merge_col, right_name, right, err)
+    else:
+        err = [(
+            'Missing merge column keyword argument(s). Must supply either `on`'
+            ', or both `left_on` and `right_on` keyword arguments'
```

Comment:
```suggestion
            ' or both `left_on` and `right_on` arguments.'
```

Reply:
But they are kwargs tho?

Reply:
this is just an editorial thing because you already say it in the prior sentence

Reply:
what about...

`
            'Missing merge column designators. Must supply either `on`'
            ' or both `left_on` and `right_on` keyword arguments'.
`

Reply:
Ohh ok